app/forms/QuestionnaireOccupationInfoForm.scala

```diff
@@ -24,17 +24,15 @@ import play.api.i18n.Messages
 
 object QuestionnaireOccupationInfoForm {
 
-  val skipValues = Seq("Unemployed but seeking work", "Unemployed", "none", "Unknown") // none is a value to denote that the field is empty
-
-  val parentsOccupationFormatter = new Formatter[Option[String]] {
+  val employedDependentFormatter = new Formatter[Option[String]] {
     override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], Option[String]] = {
       val check = data.get("employedParent")
-      val value = data.get("parentsOccupation")
+      val value = data.get(key)
```

Comment:
It might simplify things if the empty strings are dealt with before the pattern match